TMDbLib/Objects/Search/TvSeasonEpisode.cs

```diff
@@ -32,6 +32,9 @@ public class TvSeasonEpisode
         [JsonProperty("production_code")]
         public string ProductionCode { get; set; }
 
+        [JsonProperty("runtime")]
+        public int Runtime { get; set; }
```

Comment:
doesn't this need to be nullable? I think future episodes don't have a runtime

Reply:
You are right @cvium. Fixed 👍🏻